### DISCUSSION TITLE
umi3.5.34版本。默认的配置history:{type:'browser'}。url发生变化之后，页面无法更新。但是改成history:{type:'hash'}，页面会更新。

### DISCUSSION THREAD

**Discussion by shanzheng1991** [umijs/umi#9636] [General]
![image](https://user-images.githubusercontent.com/38523920/198244536-4661c8ef-5903-482a-b0b1-af8848f14fac.png)
browserHistory 与hashHistory 区别看了一下。没找到解决办法。

**Comment by yuanliu147** [comment]
这么久了没人处理吗？

**Comment by yuanliu147** [comment]
项目从 3.4.22 升级到 3.5.41 后也会出现 history.push 跳转后页面不刷新的情况

**Comment by fz6m** [comment]
umi 3 不维护了，建议升级到 umi 4 。
